src/tests/functional/inference_engine/ngraph_reshape_tests.cpp

```diff
@@ -74,18 +74,13 @@ TEST_F(NGraphReshapeTests, ReshapedDynamicShapeLayout) {
 
     CNNNetwork cnnNetwork(ngraph);
     ASSERT_EQ(Layout::NCHW, cnnNetwork.getInputsInfo()["A"]->getLayout());
-    IE_SUPPRESS_DEPRECATED_START
-    ASSERT_TRUE(cnnNetwork.getInputsInfo()["A"]->getInputData()->isDynamic());
-    IE_SUPPRESS_DEPRECATED_END
```

Comment:
should not we check that inputs data shape is filled with zeros?

Reply:
we also need to add test that such model cannot be loaded by plugins in old API and check inside the InferneceEngine::Core - because user need to reshape such dynamic models to static shapes first.

Reply:
Fixed. Added check for zeros in this test. Also added assertion to LoadNetwork/QueryNetwork